src/server/user-controller.js

```diff
@@ -183,7 +183,7 @@ class UserController {
 
   registerMiddleware(router) {
     router.use(session({
-      name: 'SESSION_ID',
+      name: `SESSION_ID_${this.serverConfig.sessionCookieDomain.toUpperCase()}`,
```

Comment:
I would prefer to have the whole session cookie name as a server config value, then it the consumer doesn't have to think about the internal string concatenation. The lambdas have the session cookie name already configurable. So in the future we could have `ELMU_SID_STAG`, `OMA_SID_INT` etc.

Reply:
Okay, will do, I found another small issue and will fix that one as well.